libafl_libfuzzer/build.rs

```diff
@@ -65,52 +65,11 @@ fn main() {
 
     let mut lib_path = custom_lib_dir.join(std::env::var_os("TARGET").unwrap());
     lib_path.push("release");
-    lib_path.push("libafl_libfuzzer_runtime.a");
-
-    // // TODO this is definitely not compat with macOS/Windows...
-    if cfg!(feature = "whole-archive") {
-        use std::path::Path;
-        let target_libdir = Command::new("rustc")
-            .args(["--print", "target-libdir"])
-            .output()
-            .expect("Couldn't find rustc's target-libdir");
-        let target_libdir = String::from_utf8(target_libdir.stdout).unwrap();
-        let target_libdir = Path::new(target_libdir.trim());
-
-        let rust_lld = target_libdir.join("../bin/rust-lld");
-        let rust_ar = target_libdir.join("../bin/llvm-ar"); // NOTE: depends on llvm-tools
```

Comment:
You can also remove this again :smile: 
https://github.com/AFLplusplus/LibAFL/blob/f7c94f9a8537dc4a8d91b2bd13e57171bc099e4a/.github/workflows/build_and_test.yml#L117

Reply:
aha ok, thanks!